store/src/types/prefix/mutable.rs

```diff
@@ -7,7 +7,7 @@ use crate::{
 
 /// Wraps an mutable KVStore with a prefix
 pub struct MutablePrefixStore<'a, DB> {
-    pub(crate) store: KVStoreMut<'a, DB>,
+    pub(crate) store: KVStoreMut<'a, DB>, //TODO: does this need to own the store?
```

Comment:
Yes. In next PR you will understand it better. Maybe. Not sure

Reply:
:+1: